spec/inverted-index-test.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+    //A test suite to read book data
+describe("Inverted Index Suite", function(){
+    let newIndex, books, emptybook, sampleString;
+        
+    beforeEach(function(){
+        //Create an instance of the Index class
+        newIndex = new InvertedIndex();
+        
+        //Test files to be used in the Index
+        books = [
+            {"title": "Heroku",
+            "text":"You will be asked to enter your Heroku credentials the first time you run a command; after the first time, your email address and an API token will be saved"
+            },
+            {"title": "Coveralls",
+            "text": "See the latest code-coverage statistics on all of your repositories including the total percentages covered and the lines covered."
+            }         
+        ];
+
+        emptybook = [
+            {}
+        ];
+
+        sampleString = "As &you can see here, we have defined *the function, useCounter(), as the target of the self-executing function %block.";
+    });
+
+    describe("Class Inverted Index Class", function() {
+        it("Inverted Index should be a class", function() {
+            expect(typeof(newIndex)).toBe("object");
```

Comment:
Not sure this if this is the best way to check if `newIndex` is a `class`.
http://www.w3schools.com/js/js_object_definition.asp
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof